dnase/trackhub/update_tracks.bash

```diff
@@ -213,6 +213,7 @@ cp -R assets/${hub_type}/. ${hub_target}
 
 for i in "${genome_array[@]}"; do
```

Comment:
what does this loop do?

Reply:
This loop is building the genomes.txt file for the hub.  The members of genome_array are the various genomes supported by the hub, like hg38 and t2t.

Reply:
add directly as a comment